src/core/interfaces.ts

```diff
@@ -397,7 +416,12 @@ export interface CoreChartProps
    * An object that contains all of the localized strings required by the component.
    * @i18n
    */
-  i18nStrings?: CartesianI18nStrings & PieI18nStrings & CoreI18nStrings;
+  i18nStrings?: CartesianI18nStrings & PieI18nStrings & CoreI18nStrings & ChartSeriesMarkerI18n;
+  /**
+   * Specifies the options for each item in the chart.
+   * @param id the item id. Can be the id of a series or a point.
+   */
+  getItemProps?: (id: string) => ChartItemOptions;
```

Comment:
Two suggestions here:
1. Rename to `getItemOptions` to be inline with Highcharts conventions and interface name
2. Update arguments to `{ itemId: string }` type to allow for more options here in the future (name, etc.)